src/main/resources/init_db.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE books (
+    id INT PRIMARY KEY AUTO_INCREMENT,
+    title VARCHAR(255),
+    price INT
```

Comment:
Using `INT` for the `price` column may not be suitable for storing monetary values, especially if you need to handle cents or fractional values. Consider using `DECIMAL` or `NUMERIC` data types for better precision with monetary values.